org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/inlayhint/LSPLineContentCodeMining.java

```diff
@@ -92,16 +92,18 @@ public void setLabel(final @Nullable String label) {
 
 	@Override
 	protected CompletableFuture<@Nullable Void> doResolve(ITextViewer viewer, IProgressMonitor monitor) {
-		if (wrapper.isActive() && canResolveInlayHint(wrapper.getServerCapabilities())) {
```

Comment:
why do we remove the `wrapper.isActive()` condition?

Reply:
for me the question why is it there. this pattern is not used consistently. in `LSPCodeMining.doResolve` for example this check is not performed. so either it's obsolete or missing in the other location

Reply:
Ok I re-added the check and added a comment about it.